src/webapp/hooks/userHooks.ts

```diff
@@ -69,3 +78,38 @@ export function useSaveUsersOrgUnits(props: UseSaveUsersOrgUnitsProps) {
 
     return { saveUsersOrgUnits };
 }
+
+export const useExportUsers = (props: UseExportUsersProps) => {
+    const { onSuccess, columns, filterOptions, orgUnitsField } = props;
+
+    const { compositionRoot } = useAppContext();
+    const snackbar = useSnackbar();
+    const loading = useLoading();
+
+    const exportUsers = React.useCallback(
+        (name: string, format: AllowedExportFormat, isEmptyTemplate = false) => {
```

Comment:
Let's be explicit about the type to help typescript infer the type (see isEmptyTemplate: any in the image)

![image](https://github.com/EyeSeeTea/user-extended-app/assets/461124/829ea5c8-7851-4331-a0c2-2eaac578a87a)

`(name: string, format: AllowedExportFormat, isEmptyTemplate: boolean)`